### PR TITLE
Chaging the README to markdown so that It's easier to read.

### PR DESCRIPTION
**Pull request by haroon-sheikh**
Also added build status icon and a link to maven repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# hsac-fitnesse-fixtures
+[![Build Status](https://travis-ci.org/fhoeben/hsac-fitnesse-fixtures.svg?branch=master)](https://travis-ci.org/fhoeben/hsac-fitnesse-fixtures) [![Maven Central](https://img.shields.io/maven-central/v/nl.hsac/hsac-fitnesse-fixtures.svg?maxAge=300)](http://search.maven.org/#search|ga|1|g:"nl.hsac")
```
Comment (fhoeben) [inline README.md]: Shouldn't the maven search url use: `a:"hsac-fitnesse-fixtures"`, instead of searching by group?

**Comment by fhoeben** [comment]
What's the advantage of renaming the LICENSE file?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+This project assists in testing (SOAP) web services and web applications by providing an application to define and run tests. To this end it contains a baseline installation of FitNesse (an acceptance testing wiki framework) and some FitNesse fixture (base) classes.
+
+The fixtures provided aim to assist in testing (SOAP) web services and web applications (using Selenium) minimizing the amount of (custom) Java code needed to define tests.
+
+The baseline Fitnesse installation offers the following features:
+* Ability to easily create a standalone (no JDK or Maven required) Fitnesse environment.
+* Run Fitnesse tests on a build server, reporting the results in both `JUnit XML` format and `HTML`.
```
Comment (fhoeben) [inline README.md]: I would not format the 2 formats here

File: README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* Fitnesse installation for fixture developers containing:
+    - the fixture base classes (and Selenium drivers for `Chrome`, `Internet Explorer`, `Edge` and `PhantomJs`),
```
Comment (fhoeben) [inline README.md]: These concepts seem a bit strange for code formatting. Maybe italics would be better?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+A zip file containing the latest release version of this project can be downloaded from the [Releases](https://github.com/fhoeben/hsac-fitnesse-fixtures/releases).
+A similar zip file containing the latest `snapshot` (i.e. not released but based on the most recent code) version is published as part of the automated build of this project at http://fhoeben.github.io/hsac-fitnesse-fixtures/hsac-fitnesse-fixtures-snapshot-standalone.zip.
+
+## To run the tests on a build server:
+Have the build server checkout the project and execute `mvn clean test-compile failsafe:integration-test`.
+The result in JUnit XML results can be found in: target/failsafe-reports (most build servers will pick these up automatically)
+The HTML results can be found in: `target/fitnesse-results/index.html`
+
+The Fitnesse suite to run can be specified by changing the value of the `@Suite` annotation in `nl.hsac.fitnesse.fixture.FixtureDebugTest`, or (preferably) by adding a system property, called fitnesseSuiteToRun, specifying the suite to run to the build server's mvn execution.
```
Comment (fhoeben) [inline README.md]: The system property should be formatted as code, don't you think?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+The Selenium configuration (e.g. what browser on what platform) to use when testing websites can be overridden by using system properties (i.e. seleniumGridUrl and either seleniumBrowser or seleniumCapabilities).
```
Comment (fhoeben) [inline README.md]: I believe these system property should be formatted as code as well.

**Comment by haroon-sheikh** [comment]
@fhoeben I've made suggested formatting changes. 
Also the reason I renamed to LICENSE was because that's what github adds by default. Also the standard is LICENSE or LICENSE.md.

**Comment by fhoeben** [comment]
The fact that GitHub has a default is not a very convincing argument to change it now. I added the file to ensure the project can be released to Maven central. They are fine with the license in its text form, so unless there is some problem that will be addressed by the change I won't change it.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+directory and as `...-standalone.zip` file in the target directory. It can be distributed by just copying the wiki directory or by copying and extracting the zip file to a location without spaces in its own name, or in its parent's names).
+This standalone installation can be started using `java -jar fitnesse-standalone.jar` from the wiki directory (or directory where the _standalone.zip_ was extracted).
+
+A zip file containing the latest release version of this project can be downloaded from the [Releases](https://github.com/fhoeben/hsac-fitnesse-fixtures/releases).
+A similar zip file containing the latest *snapshot* (i.e. not released but based on the most recent code) version is published as part of the automated build of this project at http://fhoeben.github.io/hsac-fitnesse-fixtures/hsac-fitnesse-fixtures-snapshot-standalone.zip.
+
+## To run the tests on a build server:
+Have the build server checkout the project and execute `mvn clean test-compile failsafe:integration-test`.
+The result in JUnit XML results can be found in: target/failsafe-reports (most build servers will pick these up automatically)
```
Comment (fhoeben) [inline README.md]: Given that the location of the HTML reports are formatted as code, I suggest we also format this directory as such.

Comment (haroon-sheikh) [inline README.md]: done

**Comment by haroon-sheikh** [comment]
I've reverted back to LICENSE.txt

**Comment by lummyare** [comment]
Hi Haroon,
I just cloned this repo and its so perfect to what i have been looking for. I just got a job as Automation tester using Appium/cucumber/ruby/ios/android/pom framework. I am a Calabash specialist But i need you to support me at least for a month to get my feet on the job which i started on Monday. Please I will be making monthly payment towards your support on my role please kindly email me on seunare@me.com to discuss better if you are keen and interested in helping me out. Thank you.

**Pull request merged**